apps/dokploy/templates/wordpress/docker-compose.yml

```diff
@@ -1,24 +1,29 @@
-version: "3.8"
 services:
   wordpress:
-    image: wordpress:5.8.3
+    image: wordpress:6.6.1
+    restart: always
+    depends_on:
+      - db
     environment:
       WORDPRESS_DB_HOST: db
       WORDPRESS_DB_USER: exampleuser
-      WORDPRESS_DB_PASSWORD: examplepass
+      WORDPRESS_DB_PASSWORD: ${MYSQL_PASSWORD}
       WORDPRESS_DB_NAME: exampledb
     volumes:
       - wordpress_data:/var/www/html
 
   db:
-    image: mysql:5.7.34
+    image: mariadb:11.5.2
+    restart: always
+    depends_on:
+      - db
```

Comment:
I got a error and is because this 
![Screenshot 2024-09-03 at 8 43 38 PM](https://github.com/user-attachments/assets/9530a507-3fe6-4ef7-bfa7-628faee7f419)